src/07_slices.py

```diff
@@ -12,26 +12,30 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+x, = a[1:2]
```

Comment:
Not a problem, more like an FYI. When you're looking for a single element, rather than a subset of elements, the slice syntax isn't needed, you can just address the index directly, just like JS. So, x = a[1]